src/Event.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System;
+
+namespace SpacetimeDB
+{
+    public interface IEventContext { }
+
+    public interface IReducerArgs : BSATN.IStructuralReadWrite
```

Comment:
It used to have separate `IReducerArgs` per generated module, with base `IReducerArgsBase` in the library, to make sure that user doesn't accidentally mix up reducer types from different modules.

Is this no longer a risk? Are there any public APIs that rely on `IReducerArgs`?

Reply:
It's only used with InternalCallReducer now, and we generate those calls in the table handles.

Reply:
I removed it after its main functionality moved to the generated DbConnection.Dispatch to fit the RemoteReducers; the interface was left empty.